test/old/build.js

```diff
@@ -1,17 +1,27 @@
 const fs = require('fs')
```

Comment:
any reason this file cant be killed?

Reply:
This file creates the old test.html page if you want to keep that workflow ( npm run dev and go to localist:8080/test.html )